sample-applications/blazing-coffee/BlazingCoffee/Client/Theme/main.scss

```
@@ -15,10 +15,24 @@
     color: #ffa600;
 }
 
-html body{
-    overflow: hidden;
+/* 
+    We aim to contain the app in a drawer and make the main content 
+    scrollable. The column menu of the grid is shown on the body of the application
+    and there might be occassions in which it spreads outside of the main content,
+    we should make the body scrollable but since we already have a scroll for the main
+    content, two scrollbars next to each other do not lead to good UX.
+*/
+html body {
+    overflow-y: scroll;
+    /*scrollbar-width: none;*/ 
```

```suggestion
    scrollbar-width: none;
```
Looks like we actually want this to be uncommented.